### ISSUE TITLE
Document CSS selector usage

### ISSUE THREAD

**Issue by Schahen** [JetBrains/compose-jb#1413]
Currently all css selector helpers are public and global which does't make much sense so we need to hide anything that is possible to hide and document selectors and their usage.

**Comment by Queatz** [comment]
I still to this day cannot figure out how to add a `:hover` selector to a plain old `Div({
                                style { })

**Comment by dzikoysk** [comment]
@Queatz did you find out? 😅

**Comment by dzikoysk** [comment]
Oh right, because it's translated directly to the `style` attribute, not an autogenerated class assigned to this attribute 🤦 Thanks :)

Too much React I guess, although it'd be probably a decent enhancement if it'd work the same way - way simpler than doing it via StyleSheets in most cases.

**Comment by dzikoysk** [comment]
Yup, I was experimenting with web target to find something relatively similar to React, but eventually I kinda gave up for now as it's a little bit too "raw" for now. I'll try to give it a chance one more time once I'll find some extra time to play with it.

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-6029">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
